src/components/breadcrumb-bar/index.tsx

```diff
@@ -34,35 +35,71 @@ function BreadcrumbBar({
 		})
 	// Now that we're sure all links are relative,
 	// we can render the breadcrumb links
+
+	const structuredData = [
+		{
+			'@context': 'https://schema.org',
+			'@type': 'BreadcrumbList',
+			itemListElement: links.map((link, index) => ({
+				'@type': 'ListItem',
+				position: index + 1,
+				name: link.title,
+				item: link.url
+					? `https://developer.hashicorp.com${link.url}`
+					: undefined,
+			})),
+		},
+	]
+	const stringifiedStructuredData = JSON.stringify(structuredData)
 	return (
 		<nav aria-label="Breadcrumb" className={s.root}>
+			<Head>
+				<script
+					type="application/ld+json"
+					dangerouslySetInnerHTML={{ __html: stringifiedStructuredData }}
+				/>
+			</Head>
 			<ol className={s.listRoot}>
-				{links.map(({ title, url, isCurrentPage }) => {
-					const cleanTitle = title.replace(/<[^>]+>/g, '')
-					const Elem = url ? InternalLink : 'span'
+				{links.map(({ title, url, isCurrentPage }, i) => {
 					return (
-						<Text
-							asElement="li"
-							className={s.listItem}
-							key={`${cleanTitle}_${url}`}
-							size={100}
-							weight="medium"
-						>
-							<Elem
-								className={s.breadcrumbText}
-								href={url}
-								aria-current={isCurrentPage ? 'page' : undefined}
-							>
-								{cleanTitle}
-							</Elem>
-						</Text>
+						<Breadcrumb
+							key={`${title}_${url}`}
+							title={title}
+							url={url}
+							isCurrentPage={isCurrentPage}
+						/>
 					)
 				})}
 			</ol>
 		</nav>
 	)
 }
 
+export interface BreadCrumbProps {
+	title: string
+	url: string
+	isCurrentPage: boolean
+}
+export const Breadcrumb: React.ComponentType<BreadCrumbProps> = ({
+	title,
+	url,
+	isCurrentPage,
+}) => {
+	const cleanTitle = title.replace(/<[^>]+>/g, '')
+	const Elem = url ? InternalLink : 'span'
+	return (
+		<Text asElement="li" className={s.listItem} size={100} weight="medium">
+			<Elem
+				className={s.breadcrumbText}
+				href={url}
+				aria-current={isCurrentPage ? 'page' : undefined}
+			>
+				<span itemProp="name">{cleanTitle}</span>
+			</Elem>
+		</Text>
+	)
+}
```

Comment:
A couple of questions/notes here:

- What is the purpose of these changes overall?
- Do `Breadcrumb` and `BreadCrumbProps` need to be exported? I'm not sure we'll want these to be useable outside of this component.
- The `itemProp` on the `span` isn't needed, so I think that line can be changed back to just `{cleanTitle}`.

Reply:
I previously exported these new components for a local test, but that file has since been deleted.

I'll go and revert these changes.